app/mailers/rent_mailer.rb

```diff
@@ -2,6 +2,6 @@ class RentMailer < ApplicationMailer
   def notification_mail(rent)
     @rent = rent
     @user = @rent.user
-    mail(to: @user.email, subject: 'Your rent has_been registered')
+    mail(to: @user.email, subject: t('rent_notification_mail.subject'))
```

Comment:
We only need to translate email subject, [we can use #with_locale method](https://translation.io/blog/rails-i18n-with-locale)
```rb
   I18n.with_locale(@user.locale) do
      mail(to: @user.email, subject: I18n.t('rent_notification_mail.subject'))
    end
```